tests/helpers.js

```diff
@@ -1364,4 +1572,64 @@ describe('helpers', function () {
     assert.equal(expect, result.type);
     done();
   });
+
+  it('collectSuffixExtensions - no extensions', function () {
+    var ruleset = gonzales.parse(['',
+      '.a {\n',
```

Comment:
I thought this was easier to read than including the already-parsed gonzales AST.

I personally prefer array joins over string concatenation. This can be cleaned up slightly by including the newlines within the `join`